### PR TITLE
algernon 0.85

### LINKED ISSUE #43897: algernon failed to build on 10.9.5, 10.11
`algernon` is failing to build for me. Looks like it's missing a dependency.

```
==> go build -o algernon
	/usr/local/Cellar/go/1.5.1/libexec/src/github.com/klauspost/crc32 (from $GOROOT)
	/private/tmp/algernon20150913-15605-12uy9xg/algernon-0.84/src/github.com/klauspost/crc32 (from $GOPATH)
src/github.com/fatih/color/color.go:9:2: cannot find package "github.com/mattn/go-isatty" in any of:
	/usr/local/Cellar/go/1.5.1/libexec/src/github.com/mattn/go-isatty (from $GOROOT)
	/private/tmp/algernon20150913-15605-12uy9xg/algernon-0.84/src/github.com/mattn/go-isatty (from $GOPATH)
```

### PR DESCRIPTION
**Pull request by xyproto**
Update to 0.85.

Also resolves #43897 and #43898.

### CODE REVIEW

File: Library/Formula/algernon.rb [hunk 1]
```diff
@@ -3,62 +3,59 @@
 class Algernon < Formula
   desc "HTTP/2 web server with Lua support"
   homepage "http://algernon.roboticoverlords.org/"
-  url "https://github.com/xyproto/algernon/archive/0.84.tar.gz"
-  sha256 "bc6c9899744ae50736dee1cea1e6c3228d89f8ccf436eb22694664b828e927fc"
+  url "https://github.com/xyproto/algernon/archive/0.85.tar.gz"
+  sha256 "c30380d9781166a6cc6297210e70c742c5a6c91bbbbdbf223783db1ffa1aa7e6"
   head "https://github.com/xyproto/algernon.git"
 
-  bottle do
```
Comment (dunn) [inline Library/Formula/algernon.rb]: Leave the bottle block; our bot updates it automatically!

**Comment by xyproto** [comment]
Added the bottle block back in.

**Comment by xyproto** [comment]
Fixed an issue with `assert_match`.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/825031cc01cbc1b51e11e0a497fdab51a823867b. Thank you for your contribution to Homebrew @xyproto!

**Comment by xyproto** [comment]
Thank you, @DomT4

**Pull request closed without merging**